builders.py

```diff
@@ -87,9 +87,7 @@ def start(self):
         echo "Checking for simplejson or json";
         "$VEPYTHON" -c 'import json' 2>/dev/null || "$VEPYTHON" -c 'import simplejson' ||
                     "$VE/bin/pip" install --no-index --download-cache="$PWD/.." --find-links="$PKG_URL" simplejson || exit 1;
-        echo "Checking for sqlite3, including pysqlite3 on Python 2.5";
         "$VEPYTHON" -c 'import sqlite3, sys; assert sys.version_info >= (2,6)' 2>/dev/null ||
-                    "$VEPYTHON" -c 'import pysqlite2.dbapi2' ||
                     "$VE/bin/pip" install --no-index --download-cache="$PWD/.." --find-links="$PKG_URL" pysqlite || exit 1
```

Comment:
If old lines 92 and 93 are removed then line 91 is only asserting that the python version is at least 2.6 and that'll not fail too. So, old lines 91 and 93 can be removed.